src/data-services/pcex-backend.js

```diff
@@ -43,4 +43,26 @@ export function testDataRequest(area) {
     }
   )
   .then(response => response.data);
-}
\ No newline at end of file
+}
+
+export function testLongTermAverageDataRequest(area) {
+    // accept only a specified area -
+    // always gets data from the same file.
```

Comment:
This isn't correct - this API gets data from several files. Maybe change "same file" to "same set of files".